services/QuillLMS/app/workers/populate_annual_vitally_worker.rb

```diff
@@ -21,7 +21,7 @@ def perform
   end
 
   def schools_to_sync
-    School.select(:id).distinct.joins(:users).where('users.role = ?', 'teacher')
+    School.select(:id).distinct.joins(:users).merge(User.teacher)
```

Comment:
Never seen this before, cool!

Reply:
It was new for me, too!  I was trying to figure out how to use a `scope` with `joins`, and Google surfaced this technique.  It feels a lot cleaner to me.

Reply:
Cool.  I haven't seen this either.